src/pages/Preview/containers/Preview.js

```diff
@@ -2,15 +2,24 @@ import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import ReactPlayer from 'react-player';
 import { getSongByTitleId } from 'utils/ktvQueries';
+import { useGlobalContext } from '../../../global/context';
 
 const Preview = ({ match }) => {
   const {
     params: { songName },
   } = match;
+  const globalContext = useGlobalContext();
+  const [karaokeState, karaokeDispatch] = globalContext.karaoke;
```

Comment:
The following changes made the dependencies array error go away, adding after this line: `const { playSong } = karaokeState;`